src/libs/uploads.ts

```diff
@@ -31,7 +86,7 @@ export const useUploader = () => {
           active: true,
           files: update.files,
           totalFiles: update.files.length,
-          totalBytes: _.reduce((total, file) => total += file.size, 0, update.files)
+          totalBytes: update.files.reduce((total, file) => total + file.size, 0),
```

Comment:
SonarCloud complained about the assignment to `total` (`total +=`). Since I was changing this line anyway, removed Lodash.

Reply:
I thought the coding guideline the UI Working Group voted for was favouring Lodash over native functions.

Reply:
I thought it was "avoid mixing native and Lodash". This is the only usage of Lodash in this module, so I don't see a need to depend on it here.

Reply:
That _and_ "prefer Lodash FP functions over native functions"

https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide#lodash

> This is the only usage of Lodash in this module, so I don't see a need to depend on it here.
I don't think we've been basing the decision on whether to depend on a library (that's already depended on by other modules), on how many times it's used in a given module so I don't see why we should here.

Reply:
Sadly, that was the decision. I would prefer the guideline to be understandable code that works (which I think both lodash and native functions satisfy, though native functions are typically more familiar to folks).

Reply:
> native functions are typically more familiar to folks

Most of the folks voted for Lodash so I'd think that it's the opposite.

That being said, I'm fine with changing the guideline, especially given the new information @nawatts shared about improved type inference. I'd recommend posting on #terra-ui and/or bringing it up at a UIWG meeting so folks are looped in.